File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>


int main(void)

{   int c;
    float n;     
 //promting for an answer, c - amount of coins, n - change owed
    do
   { 
     n = get_float ("Change owed:");
   }
    
   while (n < 0);
   
   n = n * 100;
   n = round(n);
   
   while (n > 0)
   {
       if (n >= 25)
       {
           n = n - 25;
           c = c + 1;
       }
       else if (n >= 10)
       {
           n = n - 10;
           c = c + 1;
       }
       else if (n >= 5)
       {
           n = n - 5;
           c = c + 1;
       }
       else if (n >= 1)
       {
           n = n - 1;
           c = c + 1;
       }
   }
   printf("Coins Used: %d\n", c);
}   

File: cash.py
from cs50 import get_float

#Promting for an answer, n - change owed, c - amount of coins
while True:
    n = get_float("Change owed: ")
    if n >= 0:
        break

#Converting to cents
n = round(n * 100)
c = 0
while n > 0:
    if n >= 25:
        n -= 25
        c += 1
    elif n >= 10:
        n -= 10
        c += 1
    elif n >= 5:
        n -= 5
        c += 1
    elif n >= 1:
        n -= 1
        c += 1

#Printing number of coins used
print(c)